{
  "Faults": [
    {
      "latitude": 50.77144926737719,
      "type": "LV",
      "reference": "KG2366",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-03T16:05:00.000+0000",
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ee5618aa-f9eb-434e-ae83-2cb49430eddc",
      "minLongitude": -1.9124,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9032323986235646,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-03T16:05:00.000+0000",
      "estimatedRestoration": "2022-12-04T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KG2366'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KG2366",
      "maxLongitude": -1.8935,
      "minLatitude": 50.7705,
      "name": "BH10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH10 7LF",
        "BH10 7LJ",
        "BH10 7LL",
        "BH10 7LN",
        "BH10 7LP",
        "BH10 7LW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9124,
              50.7781
            ],
            [
              -1.8935,
              50.7781
            ],
            [
              -1.8935,
              50.7705
            ],
            [
              -1.9124,
              50.7705
            ],
            [
              -1.9124,
              50.7781
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-03T16:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7781,
      "etrUpdate": true
    },
    {
      "latitude": 51.486092823924395,
      "type": "LV",
      "reference": "KG2431",
      "etaUpdate": true,
      "affectedCustomerCount": 27,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "79dc6315-9c7e-418f-995f-f4a457fff4eb",
      "minLongitude": -0.3045,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.30181178471257886,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-03T21:20:00.000+0000",
      "estimatedRestoration": "2022-12-04T00:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KG2431'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KG2431",
      "maxLongitude": -0.2997,
      "minLatitude": 51.4849,
      "name": "TW8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "TW8 0NF",
        "TW8 0NN",
        "TW8 0NQ",
        "TW8 0NW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3045,
              51.487
            ],
            [
              -0.2997,
              51.487
            ],
            [
              -0.2997,
              51.4849
            ],
            [
              -0.3045,
              51.4849
            ],
            [
              -0.3045,
              51.487
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.487,
      "etrUpdate": true
    },
    {
      "latitude": 51.4856319342149,
      "type": "LV",
      "reference": "KG2432",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "81aa86c9-585c-45f8-a67e-796edf423d5c",
      "minLongitude": -0.3041,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.30227540104409983,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-03T21:20:00.000+0000",
      "estimatedRestoration": "2022-12-04T00:35:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KG2432'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KG2432",
      "maxLongitude": -0.2997,
      "minLatitude": 51.4849,
      "name": "TW8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "TW8 0NF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3041,
              51.4865
            ],
            [
              -0.2997,
              51.4865
            ],
            [
              -0.2997,
              51.4849
            ],
            [
              -0.3041,
              51.4849
            ],
            [
              -0.3041,
              51.4865
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4865,
      "etrUpdate": true
    }
  ]
}
